.template-helpers/variant-default-buildpack-deps.md

```diff
@@ -0,0 +1 @@
+This tag is based off of [`buildpack-deps`](https://registry.hub.docker.com/_/buildpack-deps/). `buildpack-deps` is designed for the average user of docker who has many images on their system. It, by design, has a large number of extremely common Debian packages. This reduces the number of packages that images that derive from it need to install, thus reducing the overall size of all images on your system.
```

Comment:
While we're here, can we drop `registry.` from this URL? :innocent:

Reply:
(and capitalize "docker")

Reply:
I got the rest of the `registry.` references in https://github.com/docker-library/docs/commit/c06ef55af76121c7b812d5f90562c66f56154daf :+1: